cmd/ls-main.go

```diff
@@ -93,6 +97,9 @@ EXAMPLES:
 
   8. List all contents versions if the bucket versioning is enabled.
      {{.Prompt}} {{.HelpName}} --versions s3/mybucket
+
+  9. List number of objects and total size.
+     {{.Prompt}} {{.HelpName}} --recursive --summarize s3/mybucket
```

Comment:
```suggestion
     {{.Prompt}} {{.HelpName}} --summarize s3/mybucket
```

Summarize can be done without recursive as well?

Reply:
Do you mean something like this?
```
$ ./mc ls --summarize s3/tmp
Total Size: 20 MiB
Total Objects: 7
```

Summarize can achieve the below wo/ recursive.
```bash
$ ./mc ls --summarize s3/tmp            
[2020-12-08 10:53:45 CST]     0B tmp/

Total Size: 0 B
Total Objects: 1

 $ ./mc ls --summarize s3/tmp/test.t.0.sql
[2020-11-02 15:48:26 CST]  11MiB test.t.0.sql

Total Size: 11 MiB
Total Objects: 1
```

Reply:
correct @cyliu0  but your example change is fine.